chess_engine/main.js

```diff
@@ -0,0 +1,837 @@
+"use strict";
```

Comment:
Cant' read this code at all, seems to be minified code.
Seems to be taken from https://github.com/jhlywa/chess.js/blob/master/chess.min.js

It is perfectly ok to use third party code, but why to take minified one? and why to remove copyrights?